backend/kesaseteli/applications/api/v1/views.py

```diff
@@ -229,10 +256,33 @@ def activate(self, request, *args, **kwargs) -> HttpResponse:
         list(same_persons_apps)  # Force evaluation of queryset to lock its rows
 
         if same_persons_apps.active().exists():
-            return HttpResponseRedirect(youth_application.already_activated_page_url())
+            if (
+                youth_application.need_additional_info
+                and not youth_application.has_additional_info
```

Comment:
it could also go here if `has_additional_info ` is true because the page would then show the "additional info application sent succesfully" -message

Reply:
Changed this handling a bit, see https://github.com/City-of-Helsinki/yjdh/pull/966/commits/4effd58db17f24eb1ee2aefc5ad6b34e49d36fe7#diff-5f7d2e2d645cdd84790b6b7068d32d63ad6d75dbf0c1370f44f53c449d12db3bR231-R239